modules/lambda/main.tf

```diff
@@ -57,6 +57,11 @@ resource "aws_lambda_function" "lambda" {
   runtime          = "${var.runtime}"
   description      = "${var.description} (stage: ${var.stage})"
 
+  vpc_config {
+    subnet_ids         = "${var.subnet_ids}"
```

Comment:
Because of the way vars are passed between modules, I think these will need to be forced into a list here like we did with [`alarm_actions`](https://github.com/josh-paul/xplat-terraform-modules/blob/5b7594bfc61d74fd9062e4e2def10000a07f1cb4/modules/lambda/main.tf#L109)

Reply:
Updated to the list forcing